File: prime.c
#include <stdio.h>
#include <stdlib.h>
int main(int argc, char * argv[])
{
    int N = atoi(argv[1]);
    for (int i=2; i<N;i++)
    {
        int prime = 1;
        for (int j=2; j<i; j++)
        {
            int res = i % j;
            if(res==0)
            {
                prime = 0;
                break;
            }
        }
        if (prime==1) printf("%d\n", i);
    }
    return 0;
}

File: prime.py
import sys
N = int(sys.argv[1])

for i in range(2,N,1):
    prime = True
    for j in range(2,i,1):
        res = i % j
        if res==0: 
          prime = False
          break
        #cnt = cnt + 1
    if prime: print(i)
